EventFilter/CTPPSRawToDigi/src/RawToDigiConverter.cc

```diff
@@ -343,7 +343,7 @@ void RawToDigiConverter::run(const VFATFrameCollection &coll, const TotemDAQMapp
 
           const TotemDAQMapping::TotemTimingPlaneChannelPair SWpair = mapping.getTimingChannel( totemSampicFrame.getHardwareId() );
           // for FW Version > 0 plane and channel are encoded in the dataframe
-          if ( totemSampicFrame.getFWVersion() == 0 )  // Mapping not present in HW, read from SW
+          if ( totemSampicFrame.getFWVersion() < 0x30 )  // Mapping not present in HW, read from SW for FW versions < 3.0
```

Comment:
is there a history document for this?
It may help to have it linked inline to make the version number related logic less magic.

Reply:
Not at the moment, we'll try to provide it.
Here a small mapping related summary:
FW 0x00 was the first test, without any mapping information encoded in the data format.
FW 0x2n (in particular 0x23 that was used for the data taking) are firmware versions with mapping information: the mapping in the FPGA overruled the software for on-line debugging*. Now, it is better to use the mapping in the software for easier history and debugging.
In case of future version (>0x30) the mapping will be read from the FPGA for the same reasons as above.

*: as a reminder, we had one shot. One week of data taking without any way to double check the mapping before (with data).

Reply:
Do I understand correctly, that no data actually existed for production processing with ```getFWVersion() == 0```
and also that no data exists yet with version ```>= 0x30```?
In this case everything was so far processed in the ```else // Mapping read from HW, checked by SW``` block below.
It's worth taking a look at the data with these detectors present to have a sanity check.
Please suggest an input file.

Reply:
We used FW == 0 in some test runs without beam or anyway without any physical importance.
FW >= 0x30 is only for future compatibility

The 2 FW versions used in production were:
0x23 for the 90m alignment run: 318529 for example, or anyway the whole fill 6837
0x24 for the 90m dedicated run: fills 6877, 6879, 6881, 6882, 6884, 6885, 6890, 6891, 6892
a good run is for example 319270

NOTE: FW 0x24 doesn't change the data format, it was a small bug fix

Reply:
I don't find this file on disk (or anything from TOTEM3 dataset).

Reply:
@slava77 Honestly, I wouldn't know what to answer. the files are listed on DAS and we used them to test the mapping a few weeks ago and now I'm also not able to access them.
https://cmsweb.cern.ch/das/request?view=list&limit=150&instance=prod%2Fglobal&input=file+dataset%3D%2FTOTEM3%2FRun2018B-v1%2FRAW+run%3D319176